shards/input/state.hpp

```diff
@@ -139,9 +144,12 @@ struct InputState {
     pointers.pointers.clear();
     int numDevices{};
     auto touchDevices = SDL_GetTouchDevices(&numDevices);
+    currentTouchDevice.reset(touchDevices, SDL_free);
```

Comment:
Don't need to store here, can just `DEFER({SDL_free(...)})`

Reply:
That did not work, cos outFiner.touchId is actually copying

Reply:
hmm but it's just a uint64.... somehow all my inputs were broken though If I didn't do this....
Will try to understand. I had defer before.

Reply:
Worked, I think I had different slow down issues when I tested.